tests/contrib/actions/reports/test_s3.py

```diff
@@ -0,0 +1,201 @@
+try:
+    from unittest.mock import MagicMock, patch, mock_open
```

Comment:
You can use `pytest-mock` library (and the `mocker` fixture) as you did in test_files.py to keep consistency.